app/scripts/directives/selectize/selectize.js

```diff
@@ -0,0 +1,25 @@
+(function () {
+  'use strict';
+
+  angular
+    .module('porttare.directives')
+    .directive('selectize', selectize);
+
+  function selectize() {
+    return {
+      restrict: 'A',
+      require: 'ngModel',
+      scope: {
+        options: '=',
+        config: '='
+      },
+
+      link: function(scope, elemento, attrs) {//jshint ignore:line
```

Comment:
También hay como agregar controllerAs https://github.com/johnpapa/angular-styleguide/blob/master/a1/README.md#directives-and-controlleras
y hagámoslo en el controlador, no en link...